play-json/js/src/test/scala/JsonSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Json._
+
+import org.scalatest._
+
+class JsonSpec extends WordSpec with MustMatchers {
```

Comment:
These separate tests for the JS env are due to the loose of precision with `JSON.parse` (corresponding the JS behaviour)